aspnetcore/fundamentals/websockets.md

```diff
@@ -42,28 +70,28 @@ For some apps, [gRPC on .NET](xref:grpc/index) provides an alternative to WebSoc
 
 Add the WebSockets middleware in `Program.cs`:
 
-:::code language="csharp" source="websockets/samples/6.x/WebSocketsSample/Snippets/Program.cs" id="snippet_UseWebSockets":::
+:::code language="csharp" source="websockets/samples/7.x/WebSocketsSample/Snippets/Program.cs" id="snippet_UseWebSockets":::
 
 The following settings can be configured:
 
 * <xref:Microsoft.AspNetCore.Builder.WebSocketOptions.KeepAliveInterval%2A> - How frequently to send "ping" frames to the client to ensure proxies keep the connection open. The default is two minutes.
 * <xref:Microsoft.AspNetCore.Builder.WebSocketOptions.AllowedOrigins%2A> - A list of allowed Origin header values for WebSocket requests. By default, all origins are allowed. For more information, see [WebSocket origin restriction](#websocket-origin-restriction) in this article.
 
-:::code language="csharp" source="websockets/samples/6.x/WebSocketsSample/Program.cs" id="snippet_UseWebSockets":::
+:::code language="csharp" source="websockets/samples/7.x/WebSocketsSample/Program.cs" id="snippet_UseWebSockets":::
 
 ## Accept WebSocket requests
 
 Somewhere later in the request life cycle (later in `Program.cs` or in an action method, for example) check if it's a WebSocket request and accept the WebSocket request.
 
 The following example is from later in `Program.cs`:
 
-:::code language="csharp" source="websockets/samples/6.x/WebSocketsSample/Snippets/Program.cs" id="snippet_AcceptWebSocketAsync" highlight="7":::
+:::code language="csharp" source="websockets/samples/7.x/WebSocketsSample/Snippets/Program.cs" id="snippet_AcceptWebSocketAsync" highlight="7":::
 
 A WebSocket request could come in on any URL, but this sample code only accepts requests for `/ws`.
 
 A similar approach can be taken in a controller method:
 
-:::code language="csharp" source="websockets/samples/6.x/WebSocketsSample/Controllers/WebSocketController.cs" id="snippet":::
+:::code language="csharp" source="websockets/samples/7.x/WebSocketsSample/Controllers/WebSocketController.cs" id="snippet_Controller_Connect":::
```

Comment:
I just noticed that we have two different controller examples, not sure why that's needed?

Reply:
Are you referring to versioned examples?   6.x vs 7.x?

The 6.x sample is for the 6.x version of this topic with its own code snippet regions.   When the 6.x topic reaches retirement to an archive the whole thing will be archivable.

Reply:
Eh, nvm. One is "you can use websockets with controllers", and the other is explaining HTTP/2 websockets and highlighting the specific line in the controller code.